src/components/GifButton.tsx

```diff
@@ -0,0 +1,34 @@
+// React imports
+import React from 'react';
+
+// Functional component that displays a clickable gif with a caption
+// Layout:
+// Image, Caption
+
+type Props = {
+  caption: string;
+  image: string;
+  containerStyle?: object;
```

Comment:
would just call this style, or even better would take in a `classname` prop, and append to classname on line 17